packages/runtime-core/src/renderer.ts

```diff
@@ -708,7 +708,14 @@ function baseCreateRenderer(
       // Only static vnodes can be reused, so its mounted DOM nodes should be
       // exactly the same, and we can simply do a clone here.
       // only do this in production since cloned trees cannot be HMR updated.
-      el = vnode.el = hostCloneNode(vnode.el)
+      el = hostCloneNode(vnode.el)
+      // #3072
+      // should clone custom DOM props,
```

Comment:
I think that, for reference, can we should make clear that this specific property is added by `v-model` and also used for `<progress>` elements.

"custom DOM props" makes it sound more generic than it is.

Reply:
Aha, I think there may be more custom DOM props that need to be added in the future, not just `_value`, for example, for v-show's `el._vod`, of course, we don’t need to deal with `el._vod`, but it can be used as an example.

Reply:
Yeah there might be others, so it's good to mention that all custom DOM props which are added by Vue should be included here - `_value` is one we know and can be specific about.